Where_Is_My_Wife/Assets/_Main/Scripts/UI/UnityEventText.cs

```diff
@@ -0,0 +1,20 @@
+using TMPro;
+using UnityEngine;
+
+namespace WhereIsMyWife.UI
+{
+    public class UnityEventText : MonoBehaviour
```

Comment:
This is not needed, assign text directly

Reply:
090f3bf16c54edf415f841319449c7e9225e904c